### ISSUE TITLE
final groups' lengths have a big difference

### ISSUE THREAD

**Issue by tinyhys** [tangerzhang/ALLHiC#82]
Hi dear senior,
Thank to your excellent  scripts I was able to obtain 9 groups of contigs, however, I found that group2 has a ultra big length while group9 is so short. I would appreciate a lot if you can give me some advice on the issue. Best wishes! 
group1	25337140
group2	62663953
group3	28798746
group4	43408374
group5	30450721
group6	22661883
group7	20694593
group8	24427296
group9	1003939

**Comment by tinyhys** [comment]
![image](https://user-images.githubusercontent.com/37066354/110127312-74d2bd80-7e00-11eb-89d6-de79a7aec406.png)
this is the group2 heatmap, it seems there are some error clustering in it. Could you please give some suggestions or criteria to split it into two parts and how to deal with the shortest one ?

**Comment by tangerzhang** [comment]
Yes, it seems that the group2 is a chimeric scaffold. As far as I know, juicebox can adjust the hic scaffolding. You actually can convert ALLHiC assembly to .hic and .assembly files using codes provided by PhaseGenomics and take these files to feed juicerbox. Codes can be found here (https://github.com/tangerzhang/ALLHiC/issues/68). 
Alternatively, you can find breakpoints in the heatmap based on bins in x-axis, and then manually cut this group into several scaffolds. Breakpoints are labeled in the following figure.
![breakpoint](https://user-images.githubusercontent.com/6403754/110201758-c9794580-7e9f-11eb-9314-d008371b2d26.png)

**Comment by tinyhys** [comment]
thanks for your advice! I will try to solve it~
